src/list_handler.ts

```diff
@@ -1,13 +1,24 @@
 import { APIGatewayEvent } from "aws-lambda";
 import { generateResponse } from "./utils";
-import productList from "../mocks/productList.json";
-import { Product } from "../types/product";
+import { getDbPool } from "../DB/connection";
+import { getProductListQuery } from "../DB/queries";
 
 export const getProductsList = async (event: APIGatewayEvent) => {
```

Comment:
you should have your unit tests from prev task failed after you changed implementation to db :)